src/components/WrappedSummary.jsx

```diff
@@ -0,0 +1,260 @@
+    ) {
+      return null
+    }
+    return parseFloat(document.getElementById(id).value)
+  }
+
+const isValidInput = (val) => (!isNaN(val) && val > 0 && val !== null)
+
+
+export default class WrappedSummary extends React.Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        someVar: '...'
+      };
+      this.handleCollateral = this.handleCollateral.bind(this)
+      this.handleMinting = this.handleMinting.bind(this)
+    }
+
+    handleCollateral() {
+    if (getNew("more_collateral") !== null ) {
+      this.setState({
+        someVar: getNew("more_collateral")
+      })
+    } else {
+      this.setState({
+        someVar: 0
+      })
+    }
+    }
+
+    handleMinting() {
+      if (getNew("more_gard") !== null) {
+        this.setState({
+          someVar: getNew("more_gard")
+        })
+      } else {
+      this.setState({
+        someVar: 0
+      })
+    }
+    }
+
+    render() {
+      switch (this.props.context) {
+        case 'add_collateral':
+          return <Child handler = {this.handleCollateral} someVar={this.state.someVar} transactionData={this.props.transactionData} context={this.props.context} />
+        case 'mint_gard':
+          return <Child handler = {this.handleMinting} someVar={this.state.someVar} transactionData={this.props.transactionData} context={this.props.context}/>
+      }
+    }
+  }
+
+  class Child extends React.Component {
+    render() {
+      switch (this.props.context) {
+        case 'add_collateral':
+          return (
+            <SpecificsContainer>
+                <div
+                    style={{
+                      display: 'flex',
+                      flexDirection: 'row',
+                      marginTop: 20,
+                      marginBottom: 0,
+                      justifyContent: 'space-between',
+                      alignItems: 'center',
+                    }}
+                  >
+                <div>
+                    <SpecificsTitle>{'New Collateral Added (Algos)'}</SpecificsTitle>
+                </div>
+                <div
+                style={{
+                display: 'flex',
+                flex: 1,
+                }}
+                >
+                <TransactionInput
+                    placeholder="Enter Value Here"
+                    id="more_collateral"
+                    onChange={this.props.handler}
+                  />
+                </div>
+              </div>
+                <div
+                    style={{
+                      display: 'flex',
+                      flexDirection: 'row',
+                      marginTop: 20,
+                      marginBottom: 20,
+                      justifyContent: 'space-between',
+                    }}
+                    >
+                    <div>
+                      <SpecificsTitle>{'New Collateralization Ratio'}</SpecificsTitle>
+                    </div>
+                    <div>
+                      <SpecificsValue>{
```

Comment:
The position of the elements on the righthand side of the transaction summary are not as they are currently. Please figure out a way to right-align them.

Reply:
![current_alignment](https://user-images.githubusercontent.com/43038510/173858191-07569254-c810-4a72-a1dc-2654ea39e564.PNG)
See here for comparison. This happens in both add_collateral and mint